README.md

```diff
@@ -1,56 +1,37 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Basic Webpack Setup
 
+> This is a basic webpack setup project demo.
```

Comment:
- [ ] Please add a proper project name and description to make your readme file more professional. 
For example, you can name the project `To-do list`.